packages/components/src/components/install-guide/install-instructions/Python.vue

```diff
@@ -15,7 +15,14 @@
     <section>
       <h3>Add the <code class="inline">appmap</code> package to your project</h3>
       <div>
-        <v-code-snippet clipboard-text="pip install appmap" :kind="ctaButtonType" />
+        <v-code-snippet
+          clipboard-text="pip install --require-virtualenv appmap"
+          :kind="ctaButtonType"
+        />
+        <p>
+          Note that this requires that you create and activate a
+          <a href="https://docs.python.org/3/library/venv.html">virtual environment</a>.
```

Comment:
My only question was whether it works to put links in these pages. From looking at other pages, it looks like we do this elsewhere so it's fine. @dustinbyrne  ?

Reply:
Links like this work as expected from our webviews.